README.md

```diff
@@ -33,6 +33,10 @@ conda install faiss-cpu -c pytorch
 ```
 [Here](https://github.com/facebookresearch/faiss/blob/master/INSTALL.md) you can find the above installation command.
 
+If you plan to use SPECTER, follow the set-up steps 1 and 2 described [here](https://github.com/allenai/specter#how-to-use-the-pretrained-model). Provide the SPECTER repository path in the config as `model_params.specter_dir`.
+
+If you plan to use Multifacet-Recommender, download the checkpoint files from [here](https://drive.google.com/file/d/1_mWkQ1dr_Vl121WZkbNyNMV3G_bmoQ6s/view?usp=sharing), extract it, and pass the path to `feature_vocab_file` and `model_checkpoint_dir`
```

Comment:
how do I download this using `wget`?

Reply:
I'm assuming that `model_checkpoint_dir` is `multifacet_recommender_data/mfr_model_checkpoint` in the compressed file, is this correct?